src/meilisearch/mod.rs

```diff
@@ -39,19 +40,38 @@ pub struct Meilisearch {
     env_vars: HashMap<String, String>,
 }
 
-#[derive(Debug, Clone, Copy, Eq, PartialEq)]
+/// Sets the environment of the [`Meilisearch`] instance.
+#[derive(Display, FromStr, Default, Debug, Clone, Copy, Eq, PartialEq)]
```

Comment:
I have left these changes in this PR instead of creating a `docs` and a `refactor` PR to reduce the merge-effort.